eslint-rules/lib/utils/importUtils.js

```diff
@@ -0,0 +1,100 @@
+const _ = require('lodash');
```

Comment:
This is a really hard-to-understand file..
I think adding a description to the functions can be helpful here :)

Reply:
I kind of think the method names and the two comments in the exported function are enough, which methods' names do you find confusing?

Reply:
And generally I find all lint related code hard-to-understand : ))

Reply:
BTW, it seems like you've read most if not all the code... Well done, I think... : ))

Reply:
I'm just not sure what the `addToImports` util does..

Reply:
I'm just not sure what the `addToImports` util does..

Reply:
Added docs